src/components/Main/Main.tsx

```diff
@@ -14,19 +17,23 @@ const TitleTexts: string[] = [
 ];
 
 const getTitleText = (): string => {
-  //TODO: write real logic
+  //TODO: write real logic, when filtring options will be implemented
   return TitleTexts[0];
 };
 
-function Main({handlePostClick} : { handlePostClick: (post: PostData) => void }) {
+function Main({
+  handlePostClick,
+}: {
+  handlePostClick: (post: PostData) => void;
+}) {
+  const dispatch = useAppDispatch();
+  getPosts(localStorage.jwt).then((data) => dispatch(setInitialPosts(data)));
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**